src/js/visual/image.js

```diff
@@ -436,6 +440,10 @@ ripe.Image.prototype.update = async function(state, options = {}) {
             return false;
         }
 
+        const profile = initialsProfiles
+            ? [...initialsProfiles, ...initialsSpec.profile]

```

Comment:
shouldn't this be the inverse to allow more freedom when overriding?
```suggestion
            ? [...initialsSpec.profile, ...initialsProfiles]

```

Reply:
No, since the profiles we provide in the spec should take precendence, or else it won't show in some cases